### PR TITLE
Update html5-qrcode-scanner.ts

### PR DESCRIPTION
**Pull request by Joggel72**
Add attribute type with value button to button-tags, so Firefox will not do a submit as discussed in comment https://github.com/mebjas/html5-qrcode/discussions/413#discussioncomment-2124480

### CODE REVIEW

**Comment by mebjas** [comment]
Thanks for the PR, please take a look at the review comments.

File: src/html5-qrcode-scanner.ts [hunk 1]
```diff
@@ -640,11 +641,13 @@ export class Html5QrcodeScanner {
 
         const cameraActionContainer = document.createElement("span");
         const cameraActionStartButton = document.createElement("button");
+        cameraActionStartButton.setAttribute("type", "button");
```
[the remark this answers is not in the document]
Comment (Joggel72) [reply src/html5-qrcode-scanner.ts:644]: Yes, that would be a nice solution to get every button at the same implementation state.

**Comment by mebjas** [comment]
Hi @Joggel72 - Please revert back the readme and then we can submit.

**Comment by Joggel72** [comment]
> Hi @Joggel72 - Please revert back the readme and then we can submit.

I hope I've made it correct by deleting the file.

**Comment by mebjas** [comment]
Thanks for the PR, the PR is fairly outdated.

Will send another PR with your commit (you as author) - https://github.com/mebjas/html5-qrcode/pull/588

Appreciate the contribution. Closing this one.

**Pull request closed without merging**